contrib/dist/linux/buildrpm.sh

```diff
@@ -95,48 +105,79 @@ orig_param="$@"
 #
 # usage information
 #
-usage="Usage: $0 [-b][-o][-m][-d][-u][-h] [-n <name>][-f <lf_location>][-t <tm_location>] <tarball>
+usage="Usage: $0 [-b][-o][-m][-d][-u][-s][-h] [-n name][-f lf_location][-t tm_location] tarball
+
+  -b
+             build all-in-one binary RPM only (required for all other flags to work)
+             {default: build only SRPM}
+
+  -n name  
+             name of the resulting RPM package set to "name". Requires "-b" flag.
```

Comment:
Is the quoting problematic here?  I.e., you're closing the usage string and then re-opening it.